packages/thrift/thrift.0.10.0/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "Vincent Bernardoff <vb@luminar.eu.org>"
+authors: ["The Apache Thrift Maintainers"]
+build: [ "dune" "build" "-p" name "-j" jobs ]
+homepage: "https://github.com/vbmithr/ocaml-thrift-lib"
+bug-reports: "https://github.com/vbmithr/ocaml-thrift-lib/issues"
+doc: "https://vbmithr.github.io/ocaml-thrift-lib/doc"
+depends: [
+  "ocaml" { >= "4.03.0" }
+  "dune" { >= "1.1" }
+]
+depexts: [
+  ["thrift"] {os = "archlinux"}
+  ["thrift-compiler"] {os-family = "debian"}
+  ["devel/thrift"] {os-family = "bsd"}
+]
```

Comment:
The depexts don't seem to be strictly needed. I could only find use in the examples. Is this normal?

Reply:
You are right, yes. I think it was necessary for CI to run. Anyway, it's probably not a bad idea to have thrift installed to use the library…